mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -308,19 +308,16 @@ func resourceMongoDBAtlasCluster() *schema.Resource {
 				Type:       schema.TypeSet,
 				Optional:   true,
 				Set:        HashFunctionForKeyValuePair,
-				Computed:   true,
```

Comment:
it's both basically - as [per documentation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v1/#tag/Clusters/operation/createLegacyCluster) nothing is returned and as you said it's deprecated. 

I am really not sure why we used **Computed** here originally.

Reply:
Yes agree that computed is not accurate in this case. 
I am curious if the same scenario is happening with `bi_connector_config` and `advanced_configuration` which were bringing a similar issue, no need to adjust in this PR but more for my understanding in case you looked into it.

Reply:
@AgustinBettati that's what I thought, but then something more complex happens there.
**Why? What is the difference?** The difference is that for labels we don't return anything, while for `advanced_configuration` the backend does return default value like:
```
  # advanced_configuration {
  #   # default_read_concern = ""
  #   # default_write_concern = ""
  #   fail_index_key_too_long = false
  #   javascript_enabled = true
  #   minimum_enabled_tls_protocol = "TLS1_2"
  #   no_table_scan = false
  #   oplog_min_retention_hours = 0
  #   # oplog_size_mb = 0
  #   sample_refresh_interval_bi_connector = 0
  #   sample_size_bi_connector = 0
  #   transaction_lifetime_limit_seconds = 0
  # }
```
**hence** when I do the `terraform apply` and it does a refresh and comparison it returns the error saying that `it was expecting advanced_configuration count to be 0 but the read returned count 1` (which basically mean that when we read the backend we load that configuration because its values are default).

What I've just said should apply also for `bi_connector_config` but I haven't tested that.